src/cdk/drag-drop/drag-utils.ts

```diff
@@ -51,6 +57,35 @@ export function transferArrayItem<T = any>(currentArray: T[],
   }
 }
 
+/**
+ * Copies item from one array to another. Modifying transferArrayItem from cdk utils found here
+ *
+ * https://github.com/angular/material2/blob/master/src/cdk/drag-drop/drag-utils.ts
+ *
+ * @param currentArray Array from which to clone the item.
+ * @param targetArray Array into which is cloned the item.
+ * @param currentIndex Index of the item in its current array.
+ * @param targetIndex Index at which to insert the item.
+ *
+ * PR NOTE: I followed the pattern of the other two functions here. It would be simpler and more
```

Comment:
Remove this note.

Reply:
Done, what do you think about the change in implementation the note suggested, passing the array element (currentElement) instead of passing currentIndex and currentArray?